.github/workflows/release.yml

```
@@ -0,0 +1,135 @@
+          python-version: ${{ matrix.python-version }}
+
+      - name: What OS and Python version
+        run: |
+          uname -a
+          python --version
+          which python
+      - name: install pep517 and twine
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install pep517 twine
+      - name: Build wheels
+        run: |
+          python -m pep517.build --binary --out-dir dist/ .
+      - name: Display content dist folder
+        run: |
+          ls dist/
+      - name: Install and test distribution
+        env:
+          MPLBACKEND: agg
+        run: |
+          pip install --pre --find-links dist lumispy[tests]
+          pytest --pyargs lumispy
+      - uses: actions/upload-artifact@v2
+        with:
+          path: ./dist/*.whl
+
+      - name: Publish wheels to PyPI
+        if: github.repository_owner == 'lumispy'
+        env:
+          # Github secret set in the lumispy/lumispy repository
+          # Not available from fork or pull request
+          # Secrets are not passed to workflows that are triggered by a pull request from a fork
+          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+        run: |
+          twine upload dist/*.whl --verbose
+  build_wheels_linux:
+    name: Wheels on ubuntu-latest
+    needs: create_release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: install twine
+        run: |
+          python -m pip install twine
+      - name: Build manylinux Python wheels
+        uses: RalfG/python-wheels-manylinux-build@v0.3.3
+        with:
+          python-versions: 'cp36-cp36m cp37-cp37m cp38-cp38 cp39-cp39'
+          build-requirements: 'cython'
+
+      - name: Build source distribution
+        run: |
+          pip install pep517
+          python -m pep517.build --source --out-dir sdist/ .
+      - name: Display content dist folder
+        run: |
+          ls dist/
+          ls sdist/
+      - name: Install and test distribution
+        env:
+          MPLBACKEND: agg
+        run: |
+          pip install --pre --find-links dist lumispy[tests]
+          pytest --pyargs lumispy
+      - uses: actions/upload-artifact@v2
+        with:
+          path: |
+            ./dist/*-manylinux*.whl
+            ./sdist/*.tar.gz
+      - name: Publish wheels to PyPI
+        if: github.repository_owner == 'lumispy'
+        env:
+          # Github secret set in the lumispy/lumispy repository
+          # Not available from fork or pull request
+          # Secrets are not passed to workflows that are triggered by a pull request from a fork
+          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+        run: |
+          twine upload dist/*-manylinux*.whl --verbose 
```

Depending on one of the comment above, this may need to be changed to:
```suggestion
          twine upload dist/*-py3-none-any.whl --verbose
```

---

as above, similar as in ipywidgets *.whl is directly uploaded